src/draw/lv_draw_label.h

```diff
@@ -77,6 +77,7 @@ typedef struct {
 typedef enum {
     LV_DRAW_LETTER_BITMAP_FORMAT_A8,
     LV_DRAW_LETTER_BITMAP_FORMAT_IMAGE,
+    LV_DRAW_LETTER_VECTOR_FORMAT,
 } lv_draw_glyph_bitmap_format_t;
```

Comment:
This type should be removed and `lv_font_glyph_format_t` should be used instead. (See below).